### ISSUE TITLE
Compiling perlbench ends with a core dump

### ISSUE THREAD

**Issue by sjalander** [phate/jlm#38]
Executing:

jlc -c -o dist/IO/poll.o -DSPEC -DNDEBUG -DPERL_CORE -I. -Idist/IO -Icpan/Time-HiRes -Icpan/HTML-Parser -Iext/re -Ispecrand -DDOUBLE_SLASHES_SPECIAL=0 -DSPEC_AUTO_SUPPRESS_OPENMP -D_LARGE_FILES -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64          -g -DSPEC_LINUX_X64       -DSPEC_LP64  dist/IO/poll.c

Results in:

This should not have happened!
UNREACHABLE executed at libjlm/src/frontend/llvm/llvm2jlm/instruction.cpp:52
Aborted (core dumped)

**Comment by HiVie** [comment]
Hi, Is this problem fixed? I also encountered this problem ：）

**Comment by phate** [comment]
Hi, I did not prioritize this issue so far, as it was not important for us yet. Do you need it urgently fixed? Maybe I can take a look at it these days...

**Comment by HiVie** [comment]
> Hi, I did not prioritize this issue so far, as it was not important for us yet. Do you need it urgently fixed? Maybe I can take a look at it these days...


Yes, I am trying to optimize some code with it recently, if possible, I still hope you can fix it. Thanks  :)
